usaspending_api/database_scripts/matview_generator/mv_directpayment_award_search.json

```diff
@@ -9,6 +9,12 @@
     "  awards.category,",
     "  awards.type,",
     "  awards.type_description,",
+    "  awards.generated_unique_award_id,",
+    "  CASE",
+    "    WHEN awards.fain IS NOT NULL THEN awards.fain",
+    "    ELSE awards.uri",
+    "  END AS display_award_id,",
```

Comment:
Can change display_award_id to `COALESCE(awards.fain, awards.uri)`

Reply:
Updated in other, directpayments, grant, and loan